design-proposals/determining-host-model-migratability-level.md

```diff
@@ -0,0 +1,123 @@
+This template is simply a guide. When making proposals, feel free to add and
+remove portions of this outline to best fit your specific case. There is no
+expectation that people strictly stick to this outline, but it is a good place
+to start. 
+
+# Overview
+Add a label to determine the host-model migratability level of each node 
+and use this label to configure the cluster to prefer\require node with
+high host-model migratability level when scheduling vm for the first time.
+## Motivation
+If a vm with host-model cpuModel is scheduled to a node with a unique `host-model-required-feature`
```

Comment:
For the ones that aren't familiar with the host-model mechanism, maybe it's worth adding some background. Maybe something like:

_In Kubevirt, the default CPU model is "host-model" <LINK TO LIBVIRT DOCUMENTATION>.
For host-model VMs the CPU model is being determined by the first node on which the VM lands on. This means that the first node a VM runs on can significantly impact the possible destination for future migrations._

Reply:
Done